docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -171,6 +171,24 @@ The minimal age of a query in the history before it is expired. An expired
 query is removed from the query history buffer and no longer available in
 the :doc:`/admin/web-interface`.
 
+``query.remote-task.enable-adaptive-request-size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-boolean`
+* **Default value:** ``true``
+
+Enables dynamic split of ``taskUpdateRequest`` for schemas featuring large
```

Comment:
taskUpdateRequest isn't a user-visible construct. I'd only mention that it's dynamically splitting tasks.

You can also say "large schemas." A schema is the shape of the table, and there's no need to say "schema string."

So my proposal:
"Enables dynamically splitting up server requests sent by tasks, which can prevent out-of-memory errors for large schemas."